managed-cluster-config-services/manged-cluster-config.yaml

```diff
@@ -0,0 +1,12 @@
+services:
+- hash: none
+  name: managed-cluster-config
+  path: 00-osd-manged-cluster-config.selectorsyncset.yaml 
```

Comment:
@rafael-azevedo the original path is prefixed with a `/` so I would think that is also needed here cc/ @jmelis

Reply:
:+1: